lib/index.js

```diff
@@ -116,6 +117,7 @@ function createRouter(spec, options) {
     const router = koarouter();
 
     const controllerDirectory = (options && !_.isUndefined(options.controllerDirectory)) ? options.controllerDirectory : CONTROLLER_DIRECTORY_DEFAULT;
+    const requestDebugErrorHandler = (options && !_.isUndefined(options.requestDebugErrorHandler)) ? options.requestDebugErrorHandler : REQUEST_DEBUG_ERROR_HANDLER_DEFAULT;
```

Comment:
why not just rename to `customErrorHandler`?
There's also no need to check defined, just set it to options.customErrorHandler